src/styles/index.scss

```diff
@@ -1,3 +1,144 @@
+@import './utils/variables';
+
 body {
   margin: 0;
 }
+
+img {
+  display: flex;
+  padding: 32px 19px 40px;
+  width: 160px;
+  height: 134px;
+  }
+
+.product {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #FFF;
+  width: 200px;
+  height: 408px;
+
+  &_name {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    font-style: normal;
+    padding: 0 16px;
+    width: $width-content;
+    color: $main-text-color;
+  }
+
+  &_code {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    color:$secondary-text-color;
+
+    padding: 0 16px;
+    padding-top: 4px;
+
+  }
+}
+
+.reviews {
+  display: flex;
+  justify-content: space-between;
+  width: $width-content;
+  height: 16px;
+  padding: 0 16px;
+  padding-top: 16px;
+
+  &_amount {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    color: $main-text-color;
+
+
+    text-align: right;
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+  }
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    display: flex;
+    width: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
+
+
+.price {
+  display: flex;
+  padding: 0 16px;
+  padding-top: 24px;
+  width: $width-content;
+  height: 18px;
+  justify-content: space-between;
+
+  &_text {
+    color:$secondary-text-color;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &_amount {
+    color: $main-text-color;
+    font-family: Roboto, sans-serif;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 18px;
+  }
+}
+
+.button {
+  margin: 16px auto;
+  display: block;
+  width: $width-content;
+  height: 40px;
+  border-radius: 5px;
+  background-color: #00ACDC;
+  box-sizing: border-box;
+  border: 1px solid transparent;
+
+  color: #FFF;
+  text-align: center;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 40px;
+  text-transform: uppercase;
+  text-decoration: none;
+  
+  &:hover {
+    // border-radius: 5px;
+    border: 1px solid #00ACDC;
+    background-color: #FFF;
+    box-sizing: border-box;
```

Comment:
Move box-sizing to the * selector

Reply:
<img width="471" alt="Screenshot 2024-01-04 124433" src="https://github.com/mate-academy/layout_product-cards/assets/142142904/e6a7e2fe-253a-4980-b449-4cea89730a9a">
<img width="283" alt="Screenshot 2024-01-04 124443" src="https://github.com/mate-academy/layout_product-cards/assets/142142904/9ab50acc-eb69-4020-a56b-72236bafa497">

This movement destroyed me everything.

Reply:
<img width="283" alt="Screenshot 2024-01-04 124443" src="https://github.com/mate-academy/layout_product-cards/assets/142142904/33fd9ab8-10a2-4e0b-a055-eceac783bfc3">
<img width="493" alt="Screenshot 2024-01-04 124635" src="https://github.com/mate-academy/layout_product-cards/assets/142142904/c1f3087c-e5af-4231-a77c-bc878394d670">

This movement destroyed me everything.